cmd/templates/hook.tmpl

```diff
@@ -15,9 +15,6 @@ call_lefthook()
   if lefthook{{.Extension}} -h >/dev/null 2>&1
   then
     eval lefthook{{.Extension}} $@
-  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/lefthook{{.Extension}}"
-  then
-    eval "$dir/node_modules/@arkweid/lefthook/bin/lefthook{{.Extension}} $@"
```

Comment:
The executable can be spawned directly. It is slower to spawn lefthook through `npx`. Also, removing this assumes dependency on npx, yarn, etc.

Reply:
I re-read #188 and now I understand this is required